src/impulsoetl/sisab/excecoes.py

```diff
@@ -30,10 +30,15 @@ class SisabErroCompetenciaInexistente:
     def __init__(self,exception):
         self.exception = exception 
 
-    def insere_erro_database(self,sessao,traceback_str,operacao_id,periodo_id):
+    def insere_erro_database(self,sessao,traceback_str,operacao_id,periodo_id,unidade_geografica_id):
+
+        if unidade_geografica_id is None:
+            unidade_geografica_id = '28de805e-5bdc-49c3-863c-2cf87f95e371'
+        else:
+            unidade_geografica_id = unidade_geografica_id
```

Comment:
Desnecessário se o comentário acima for resolvido.

Reply:
Corrigido no commit [f3d7765](https://github.com/ImpulsoGov/etl/pull/68/commits/f3d7765ae52e9634e2e3ab9f74ecdeb40f04e240)